LICENSE

```diff
@@ -1,24 +1,15 @@
-Copyright (c) 2012, Rotem Yaari <vmalloc@gmail.com>
```

Comment:
You'll need @vmalloc's approval on that one.

Reply:
Yes, you're right. I'll try to contact him.